lib/OpenQA/CacheService/Controller/Influxdb.pm

```diff
@@ -12,7 +12,10 @@ sub minion ($self) {
         delayed => $stats->{delayed_jobs},
         failed => $stats->{failed_jobs},
         inactive => $stats->{inactive_jobs}};
-    my $workers = {active => $stats->{active_workers}, inactive => $stats->{inactive_workers}};
+    my $workers = {
+        registered => $stats->{workers},
```

Comment:
what would happen if this happens to run with an older version of the Minion package. Thinking of users of openQA that don't use the openSUSE packages, e.g. Fedora, Debian, etc.?

Reply:
💣

Reply:
That's why i bumped the dependency.

Reply:
That's why I asked because the dependency statement may not be used by maintainers.

@AdamWill @phil-hands would the Fedora+Debian packaging cope with this change?

Reply:
I sync the Fedora package spec with the one in the repo, so we would get the dependency version bump from there.

Reply:
Thanks for the heads-up -- I'll add a versioned depends for that.

So far I've not been adding versioned depends for every dependency, but I guess I should at least add them when the versions get bumped in your dependencies.

Reply:
Having just checked, I already either have versioned depends where you do, or the versions being required are older than what is in the current release, so adding them would be a bit redundant.  I may add them anyway, just to be obviously in sync.

Reply:
ok, so we should be good. Thank you.